upup/pkg/fi/cloudup/awstasks/autoscalinggroup.go

```diff
@@ -455,13 +455,14 @@ func (v *AutoscalingGroup) RenderAWS(t *awsup.AWSAPITarget, a, e, changes *Autos
 		}
 
 		if changes.LaunchTemplate != nil {
-			// @note: at the moment we are only using launch templates when using mixed instance policies,
-			// but this might change
-			setup(request).LaunchTemplate = &autoscaling.LaunchTemplate{
-				LaunchTemplateSpecification: &autoscaling.LaunchTemplateSpecification{
-					LaunchTemplateName: changes.LaunchTemplate.ID,
-					Version:            &launchTemplateVersion,
-				},
+			spec := &autoscaling.LaunchTemplateSpecification{
+				LaunchTemplateId: changes.LaunchTemplate.ID,
+				Version:          aws.String("$Latest"),
```

Comment:
does using `$Latest` here affect the "up to date" vs "needs update" logic of `kops rolling-update cluster`?

Reply:
I made the changes based on the other PR. Thanks for noticing. Will rebase that one when this is merged.